subprojects/tooling-api/src/testFixtures/groovy/org/gradle/integtests/tooling/fixture/ToolingApiSpecification.groovy

```diff
@@ -242,4 +242,12 @@ abstract class ToolingApiSpecification extends Specification {
     protected static GradleVersion getTargetVersion() {
         GradleVersion.version(targetDist.version.baseVersion.version)
     }
+
+    /**
+     * @return the current <b>base</b> version of Gradle being used to run this test
+     */
+    protected static GradleVersion getCurrentVersion() {
+        GradleVersion.current().baseVersion
+    }
+
```

Comment:
Right, I'd introduce methods for the target version as well. I mentioned above that I'd let the method potentially rather take a `String` than a `GradleVersion` so it would look like this: `currentVersionSmallerThan('4.0')`. 

There's also the aspect of maintainability of code. I'd personally change it to the method as I think it's less error prone. I'll let @adammurdoch chime in as well as he originally reviewed the PR.